docs/adguard-browser-extension/solving-problems/logs.md

```diff
@@ -47,6 +47,8 @@ sidebar_position: 1
 
 ## Sending logs
 
+> AdGuard is committed to protecting your privacy. We strictly follow our [Privacy Policy](https://adguard.com/en/privacy/browser-extension.html), and we do not collect private information about users. Before sending your logs to the support team, please be sure to review the file as it may contain additional information that you don’t want to share. If it contains such personal information, we recommend you to delete it first.
```

Comment:
```suggestion
> AdGuard is committed to protecting your privacy. We strictly follow our [Privacy Policy](https://adguard.com/privacy/windows.html) and do not collect any private information about users. Before sending your logs to the support team, please review the file as it may contain additional information that you don’t want to share. If it contains such personal information, we recommend that you delete it first.

Reply:
ниже аналогично